README.rst

```diff
@@ -107,6 +107,8 @@ You can also retrieve it by exporting environment variables.
 
 * Required for OAuth2 Service Account Flow:
 
+.. code-block::bash
```

Comment:
Thank you for noticing this! I was just looking viewing the output of this change and noticed this code block and the one above it are still not displayed. I believe it's because there should be a space before `bash`, so:

```
.. code-block:: bash
```

Would you mind updating this line, and also the one on line 102?

Reply:
Yes, you are right.
Done.